compiler/src/iree/compiler/Codegen/Common/GPU/GPUVerifyDistribution.cpp

```diff
@@ -76,6 +76,13 @@ struct GPUVerifyDistributionPass final
               continue;
             }
 
+            // Backdoor: linalg.copy
+            if (isa<linalg::CopyOp>(op)) {
+              if (auto useDMAConfig =
+                      getLoweringConfig<IREE::GPU::UseGlobalLoadDMAAttr>(op))
+                continue;
+            }
```

Comment:
This is not done yet, right? If this part is ready for review, please add more comment and reference it to an issue.